x-pack/plugins/enterprise_search/public/applications/app_search/components/schema/components/meta_engines_conflicts_table.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { useValues } from 'kea';
+
+import {
+  EuiTable,
+  EuiTableHeader,
+  EuiTableHeaderCell,
+  EuiTableBody,
+  EuiTableRow,
+  EuiTableRowCell,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { FIELD_NAME } from '../../../../shared/schema/constants';
+import { ENGINES_TITLE } from '../../engines';
+
+import { MetaEngineSchemaLogic } from '../schema_meta_engine_logic';
+
+import { TruncatedEnginesList } from './';
+
+export const MetaEnginesConflictsTable: React.FC = () => {
+  const { conflictingFields } = useValues(MetaEngineSchemaLogic);
+
+  return (
+    <EuiTable tableLayout="auto">
+      <EuiTableHeader>
+        <EuiTableHeaderCell>{FIELD_NAME}</EuiTableHeaderCell>
+        <EuiTableHeaderCell>
+          {i18n.translate(
+            'xpack.enterpriseSearch.appSearch.engine.schema.metaEngine.fieldTypeConflicts',
+            { defaultMessage: 'Field type conflicts' }
+          )}
+        </EuiTableHeaderCell>
+        <EuiTableHeaderCell>{ENGINES_TITLE}</EuiTableHeaderCell>
+      </EuiTableHeader>
+      <EuiTableBody>
+        {Object.entries(conflictingFields).map(([fieldName, conflicts]) =>
+          Object.entries(conflicts).map(([fieldType, engines], i) => {
+            const isFirstRow = i === 0;
+            return (
+              <EuiTableRow key={`${fieldName}-${fieldType}`}>
+                {isFirstRow && (
+                  <EuiTableRowCell
+                    rowSpan={Object.values(conflicts).length}
+                    data-test-subj="fieldName"
+                  >
+                    <code>{fieldName}</code>
+                  </EuiTableRowCell>
+                )}
```

Comment:
Hopefully this isn't _too_ difficult to grok code-wise, but the goal of the first row check and `rowSpan` behavior is to essentially render this:

<img width="893" alt="" src="https://user-images.githubusercontent.com/549407/118187534-e8daa480-b3f3-11eb-8dd6-69a0531b4ad5.png">

TBH it's still less code than the original ent-search view, purely by having less custom design etc :)